.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
   test_ios:
     name: Test (x86_64-apple-ios)
-    runs-on: macos-latest
+    runs-on: macos-10.15
```

Comment:
GitHub is rolling out macOS 11 under the macos-latest label, which breaks all sorts of linking behavior https://github.com/actions/virtual-environments/issues/4060. We'll eventually need to start using cargo-dinghy for public repos like this to support macOS 11.